Scripts/demand/freight.py

```diff
@@ -55,9 +72,9 @@ def fratar(self, prod, trips, max_iter = 10):
         return trips
 
     def calibrate(self, b, n, s):
-        """Calibrate a vector
-        b = true value for base
-        n = forecast for base
-        s = forecast to calibrate
+        """Calibrate a vector\n
+        b = true value for base\n
+        n = forecast for base\n
+        s = forecast to calibrate\n
```

Comment:
Best practice: rename the variables to be more descriptive so you don't need comments here.

```   
def calibrate(self, true_value_for_base, forecast_for_base, forecast_to_calibrate):
```   

Number 5 is also a bit mystery here, it could be declared as a const: 
```   
SOME_HUMAN_READABLE_NAME=5
return numpy.where(s <  SOME_HUMAN_READABLE_NAME *n, s * b/n, s +  SOME_HUMAN_READABLE_NAME*(b - n))
```